src/server/controllers/match.controller.js

```diff
@@ -18,67 +18,59 @@ export const REJECTED = 0;
 
 // The mentee sent request to Mentor
 export function requestMentoring(req, res, next) {
-  if (req.session._id) {
-    let matchData = req.body;
-    matchData.mentee_id = req.session._id;
-    let match = new Match(matchData);
+  let matchData = req.body;
```

Comment:
**const** is better than **let** for immutable variables.
https://medium.com/javascript-scene/javascript-es6-var-let-or-const-ba58b8dcde75